src/pages/search/advanced_search/OrganizationFilters.tsx

```diff
@@ -35,6 +36,15 @@ export const OrganizationFilters = ({ topLevelOrganizationId, unitId }: Organiza
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const subUnitQuery = useQuery({
+    queryKey: [unitId],
```

Comment:
ser de aller fleste oppslag mot organization ikke har en key-prefiks `organization` elns nå, så lar denne også ligge uten for å unngå ekstra kall. Burde nok ha en full gjennomgang å legge til prefiks for alle useQueries ved et tidspunkt, og helst at alle keys da kommer fra en enum